File: PHP/which_are_in.php
<?php

    function inArray($array1,$array2){
        $result = array();
        foreach($array1 as $str1){
            foreach($array2 as $str2){
                if(strpos($str2,$str1)!==false){
                    $result[] = $str1;
                    break;
                }
            }
        } 
        
        $result = array_unique($result);
        sort($result);
        return $result;
    }

File: Python/which_are_in.py
def in_array(array1,array2):
    result = []
    for i in array1:
        for j in array2:
            if i in j:
                result.append(i)
                result = sorted(list(set(result)))
    return result

print(in_array(["arp", "live", "strong"], ["lively", "alive", "harp", "sharp", "armstrong"]))